test/libdnf/rpm/repo_fixture.cpp

```diff
@@ -0,0 +1,65 @@
+/*
+Copyright (C) 2020 Red Hat, Inc.
+
+This file is part of libdnf: https://github.com/rpm-software-management/libdnf/
+
+Libdnf is free software: you can redistribute it and/or modify
+it under the terms of the GNU Lesser General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+Libdnf is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU Lesser General Public License for more details.
+
+You should have received a copy of the GNU Lesser General Public License
+along with libdnf.  If not, see <https://www.gnu.org/licenses/>.
+*/
+
+
+#include "repo_fixture.hpp"
+
+#include <filesystem>
+
+
+void RepoFixture::setUp() {
```

Comment:
How do you want to support a multiple repository environment?

Reply:
Not sure what you mean exactly, but I didn't plan for anything like that. I've just created the class just to share (and deduplicate) the repository setup code. If there are test suites that require different repo setup, the code can be changed to accommodate for that when needed.

Reply:
I can have two repository for tests. Then I need to set which one I want to load. See dnf, hawkey unittests.

Reply:
When you have the two repos, you're free to adjust the code to make it work :slightly_smiling_face: These are just unittests, not a public API... I've just fixed the multiple duplication of the code and set up the fixture for it.